### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by RuoyuChen10** [ZS123-lang/TENET#4]
You mentioned in the paper that "TENET uses ResNet-50 pre-trained on ImageNet [5] and MS COCO [28]." I noticed that it also pre-trained on MS-COCO, does it mean that use the detector pre-trained model? E.g. ImageNet pre-trained ResNet-50, and then pretrain FRCN on MS COCO?

**Comment by ZS123-lang** [comment]
Hi, Ruoyu. Thank you for your interest. For the pretrained backbone, we follow the same approach as  [FSOD_ARPN](https://github.com/fanq15/FSOD-code). You can download in [this link ](https://dl.fbaipublicfiles.com/detectron/35857281/12_2017_baselines/e2e_faster_rcnn_R-50-C4_2x.yaml.01_34_56.ScPH0Z4r/output/train/coco_2014_train%3Acoco_2014_valminusminival/generalized_rcnn/model_final.pkl).

**Comment by RuoyuChen10** [comment]
> Hi, Ruoyu. Thank you for your interest. For the pretrained backbone, we follow the same approach as [FSOD_ARPN](https://github.com/fanq15/FSOD-code). You can download in [this link ](https://dl.fbaipublicfiles.com/detectron/35857281/12_2017_baselines/e2e_faster_rcnn_R-50-C4_2x.yaml.01_34_56.ScPH0Z4r/output/train/coco_2014_train%3Acoco_2014_valminusminival/generalized_rcnn/model_final.pkl).

Thanks, last question. For the FSOD dataset, how to get the 5-shot support set? Random select from the test set? And will the image used to generate a support prototype be used in the evaluation? Thanks!

**Comment by ZS123-lang** [comment]
No worries, asking more questions is fine. During the training process, we select support samples from the training dataset, while during the evaluation stage, we use the test dataset. The selection of support objects is not completely random and is subject to certain conditions. For example, the support objects are not selected from the query image and not chosen from the same images as much as possible.

**Comment by ZS123-lang** [comment]
> > Hi, Ruoyu. Thank you for your interest. For the pretrained backbone, we follow the same approach as [FSOD_ARPN](https://github.com/fanq15/FSOD-code). You can download in [this link ](https://dl.fbaipublicfiles.com/detectron/35857281/12_2017_baselines/e2e_faster_rcnn_R-50-C4_2x.yaml.01_34_56.ScPH0Z4r/output/train/coco_2014_train%3Acoco_2014_valminusminival/generalized_rcnn/model_final.pkl).
> 
> Thanks, last question. For the FSOD dataset, how to get the 5-shot support set? Random select from the test set? And will the image used to generate a support prototype be used in the evaluation? Thanks!

No worries, asking more questions is fine. During the training process, we select support samples from the training dataset, while during the evaluation stage, we use the test dataset. The selection of support objects is not completely random and is subject to certain conditions. For example, the support objects are not selected from the query image and not chosen from the same images as much as possible.
